.github/workflows/services-json.yml

```diff
@@ -0,0 +1,89 @@
+name: Services Validator
+
+on:
+  push:
+    paths:
+      - "plugins/rtmp-services/data/services.json"
+      - "plugins/rtmp-services/data/package.json"
+  pull_request:
+    paths:
+      - "plugins/rtmp-services/data/services.json"
+      - "plugins/rtmp-services/data/package.json"
+  schedule:
+  - cron: 0 0 * * *
+  workflow_dispatch:
+
+jobs:
+  schema:
+    name: Schema
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install & Configure Python
+        run: |
+          sudo apt install python3.9-dev
+          python3.9 -m pip install jsonschema json_source_map
+
+      - name: Validate Service JSON Schema
+        run: |
+           python3.9 CI/check-jsonschema.py plugins/rtmp-services/data/services.json plugins/rtmp-services/data/schema/service-schema-v3.json
+
+      - name: Annotate Errors
+        if: ${{ failure() && github.event_name == 'pull_request' }}
+        uses: yuzutech/annotations-action@v0.4.0
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          title: 'Service JSON Errors'
+          input: './validation_errors.json'
+
+  service_check:
+    name: Service Check
+    runs-on: ubuntu-latest
+    needs: schema
+    # todo remove my repo before un-drafting PR
+    if: (github.repository_owner == 'obsproject' ||  github.repository_owner == 'derrod') && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```

Comment:
Longer evaluations need to be wrapped in `${{ }}` last I checked, otherwise not the entire statement might be executed. I remember I had peculiar issues when I didn't wrap those.